code/modules/atmospherics/machinery/pipes/pipes.dm

```diff
@@ -83,10 +83,10 @@
 	parent = P
 
 /obj/machinery/atmospherics/pipe/Destroy()
-	QDEL_NULL(parent)
+	parent = null
```

Comment:
isnt parent the pipenet? pretty sure this wants to stay

Reply:
You may be right

Reply:
You may be right